lib/terrafying/components/letsencrypt.rb

```diff
@@ -137,7 +137,7 @@ def create_keypair_in(ctx, name, options={})
         ctx.resource :aws_s3_bucket_object, "#{key_ident}-cert", {
                        bucket: @bucket,
                        key: File.join(@prefix, @name, name, "cert"),
-                       content: output_of(:acme_certificate, key_ident, :certificate_pem) + @ca_cert,
+                       content: output_of(:acme_certificate, key_ident, :certificate_pem) + '\n' + @ca_cert,
```

Comment:
Ok, I see where you're going with this. Yeah this should work. I think I might've misunderstood the first time and made a mistake.

Reply:
As long as there isn't a new line already at the end of whatever comes out of `:certificate_pem`

Reply:
Any way to know other than trial and error? :/

Reply:
Let me see if I can test this locally

Reply:
You can run the `rspec` tests locally `bundle exec rspec`